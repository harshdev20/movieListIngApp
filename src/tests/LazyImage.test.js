import React from "react";
import { create, act } from "react-test-renderer";
import LazyImage from "../components/LazyImage";

it("should snapshot test LazyImage", () => {
  let root;
  act(() => {
    root = create(<LazyImage />);
  });

  expect(root.toJSON()).toMatchSnapshot();
});
