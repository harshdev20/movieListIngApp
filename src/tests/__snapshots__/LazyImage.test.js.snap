// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should snapshot test LazyImage 1`] = `
<span
  className=" lazy-load-image-background blur"
  style={
    Object {
      "backgroundImage": "",
      "backgroundSize": "",
      "color": "transparent",
      "display": "inline-block",
      "height": undefined,
      "width": undefined,
    }
  }
>
  <span
    className=" min-w-full lg:w-48"
    style={
      Object {
        "display": "inline-block",
      }
    }
  />
</span>
`;
